src/sdk.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+typedef enum {
+  SDK_NANO_X_1_2,
+  SDK_NANO_S_1_5,
+  SDK_NANO_S_1_6,
+  SDK_BLUE_2_2_5,
+  SDK_LAST
```

Comment:
`SDK_LAST` is misleading, as this value is "invalid", or the "count"/"numbers" of versions.

This PR did not introduce `SDK_LAST` (it was previously in `src/emulate.h`), but this PR could be the opportunity to rename this value to something such as `SDK_COUNT` (to indicate that this value is both the last one of `sdk_version_t` so it is suitable in a `for` loop, and that this value is not valid, because it is "one above the last valid version").

Reply:
Done